src/bundle/Resources/views/admin/content_type/list.html.twig

```diff
@@ -2,14 +2,14 @@
 
 <section class="container mt-4 px-5">
     <div class="ez-table-header">
-        <h4>{{ 'content_type.view.list.title'|trans({ '%identifier%': content_type_group.identifier })|desc('Content Types in %identifier%') }}</h4>
+        <p>{{ 'content_type.view.list.title'|trans({ '%identifier%': content_type_group.identifier })|desc('Content Types in %identifier%') }}</p>
```

Comment:
If it's a `p` tag doesn't it mean its parent container is not a header-like element anymore, does it?
Maybe we should revise the parent container class name?